### PR TITLE
Total number of txs in solana account

### LINKED ISSUE #10911: Add Solana total tx box to account
We do not show graph as we do not have data but we can show at least total number of transactions

Screenshot 2024-01-30 at 13 53 50

Screenshot 2024-01-30 at 13 48 46
Screenshot 2024-01-30 at 13 52 13

### LINKED ISSUE #15512: Account graph stuck in loading state when new tx arrive
**Describe the bug**
In case I am in account and I receive new tx. The graph looks like it is loading

**Info:**

-   Suite Version: 24.12.0

**How to reproduce**

1. Go to any account
1. Receive tx from somebody
1. Observe

**Expected behavior**
The graph would refresh and work 

**Screenshots**
Screenshot 2024-11-22 at 8 38 14

### PR DESCRIPTION
**Pull request by enjojoy**
## Description

Add box for all transactions amount

## Related Issue 

Resolve https://github.com/trezor/trezor-suite/issues/10911
Resolve #15512

## Screenshots:
image

### CODE REVIEW

**Comment by tomasklim** [comment]
Have you tested on account with more than 5 txs?

**Comment by enjojoy** [comment]
> Have you tested on account with more than 5 txs?

I did not, indeed, I fixed it https://github.com/trezor/trezor-suite/pull/15315/commits/3408002b6424306bd3fdffb04aeecf9549c6c8d4

File: packages/suite/src/views/wallet/transactions/components/InfoCard.tsx [hunk 1]
```diff
@@ -2,18 +2,12 @@ import { ReactNode } from 'react';
...
 import { HiddenPlaceholder, FormattedCryptoAmount, Sign } from 'src/components/suite';
 import { Account } from 'src/types/wallet';
 
-const InfoCardContent = styled.div`
-    display: flex;
-    width: 100%;
-    flex-direction: column;
-`;
-
 const Title = styled.div`
```
Comment (seibei-iguchi) [inline packages/suite/src/views/wallet/transactions/components/InfoCard.tsx:11]: Can you please also remove all of these custom styles? They are easily replaced with components

**Review by seibei-iguchi — DISMISSED** [review]

File: packages/suite/src/views/wallet/transactions/components/InfoCard.tsx [hunk 2]
```diff
@@ -72,7 +66,7 @@ export const InfoCard = (props: InfoCardProps) => {
 
     return (
         <Card minHeight={100}>
-            <InfoCardContent>
+            <Column width="100%" alignItems="flex-start">
```
Comment (seibei-iguchi) [inline packages/suite/src/views/wallet/transactions/components/InfoCard.tsx]: These props should not be needed in this case

File: packages/suite/src/views/wallet/transactions/components/TransactionSummary.tsx [hunk 3]
```diff
@@ -103,27 +79,31 @@ export const TransactionSummary = ({ account }: TransactionSummaryProps) => {
     const onSelectedRange = () => dispatch(updateGraphData([account], { newAccountsOnly: true }));
 
     return (
-        <Wrapper>
-            <Actions>
+        <Column alignItems="stretch">
+            <Row
+                margin={{ bottom: spacings.lg }}
+                justifyContent="space-between"
+                alignItems="center"
```
Comment (seibei-iguchi) [inline packages/suite/src/views/wallet/transactions/components/TransactionSummary.tsx]: Not needed

File: packages/suite/src/views/wallet/transactions/components/TransactionSummary.tsx [hunk 4]
```diff
@@ -103,27 +79,31 @@ export const TransactionSummary = ({ account }: TransactionSummaryProps) => {
...
+            >
                 <GraphRangeSelector onSelectedRange={onSelectedRange} align="bottom-left" />
                 <TransactionSummaryDropdown />
-            </Actions>
-            <ContentWrapper>
+            </Row>
+            <Column width="100%" alignItems="stretch">
```
Comment (seibei-iguchi) [inline packages/suite/src/views/wallet/transactions/components/TransactionSummary.tsx]: Width not needed

File: packages/suite/src/views/wallet/transactions/components/TransactionSummary.tsx [hunk 5]
```diff
@@ -103,27 +79,31 @@ export const TransactionSummary = ({ account }: TransactionSummaryProps) => {
...
                 {error ? (
                     <Card>
-                        <GraphWrapper>
+                        <Row height="320px" overflow="visible" alignItems="stretch">
```
Comment (seibei-iguchi) [inline packages/suite/src/views/wallet/transactions/components/TransactionSummary.tsx]: height={320}

**Review by seibei-iguchi — APPROVED** [review]

**Review by tomasklim — APPROVED** [review]

**Pull request merged**